contrib/mesos/docs/issues.md

```diff
@@ -1,34 +1,118 @@
 ## Known Issues
 
-### Pod Placement
+This page identifies significant known issues with the Kubernetes-Mesos distribution.
+
+* [General Known Issues](#general-known-issues)
+* [DCOS Package Known Issues](#dcos-package-known-issues), in addendum to the above.
+
+## General Known Issues
+
+These known issues apply to all builds of Kubernetes-Mesos.
+
+### Upgrades
+
+Upgrading your Kubernetes-Mesos cluster is currently "YMMV" status.
+One particularly nasty problem exists with respect to expressing executor (kubelet and kube-proxy) process configuration via command line flags.
```

Comment:
"nasty" is a subjective adjective. I'd me more objective like "One known problem exists with respect ..."